android/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!--   color for the app bar and other primary UI elements -->
+    <color name="colorPrimary">#FF5C5D</color>
```

Comment:
@michaeltintiuc @belov91 does beep support PWA manifest? We should match these values there

Reply:
@grgur it does, in fact it is a fully working PWA since yesterday :) the values match, except for the `FF5C5D` ones as it's used for the app heading in Android's task list.